addons/mail/static/src/models/thread.js

```diff
@@ -579,6 +579,7 @@ registerModel({
                 });
             }
             this.update(values);
+            this.messaging.messagingBus.trigger('o-thread-loaded-data', { thread: this });
```

Comment:
This doesn't seem to be used outside of `test_utils`. How about adding it via an override of thread in tests assets in order not to have it if not needed?

Reply:
Not worth the effort of an override, most of the `messagingBus.trigger` are done only for tests and are not made that way.